docs/src/notebooks/9_max_min_driving_force_analysis.jl

```diff
@@ -0,0 +1,198 @@
+
+using COBREXA
+using Tulip
+
+# Create a model of glycolysis fermentation.
+
+#md # !!! tip "Directions of reactions"
+#md #     Be careful when constructing models for MMDFA, the reaction directions in the model
+#md #     and thermodynamic data need to be consistent with the overall flux direction implied by the model.
+#md #     For example, in BiGG, `LDH_D` is written
+#md #     `lac__D + nad ⟷ h + nadh + pyr` and the associated ΔrG'⁰ is 23.6803 kJ/mol.
+#md #     For MMDFA no flux is calculated, so you need to write the reaction
+#md #     in the direction of flux, i.e. `h + nadh + pyr ⟶ lac__D + nad` with ΔrG'⁰ as
+#md #     -23.6803 kJ/mol.
+
+model = StandardModel("Glycolysis")
+
+mets = [
+    Metabolite("glc__D"),
+    Metabolite("g6p"),
+    Metabolite("f6p"),
+    Metabolite("fdp"),
+    Metabolite("dhap"),
+    Metabolite("g3p"),
+    Metabolite("13dpg"),
+    Metabolite("3pg"),
+    Metabolite("2pg"),
+    Metabolite("pep"),
+    Metabolite("pyr"),
+    Metabolite("lac__D"),
+    Metabolite("nadh"),
+    Metabolite("nad"),
+    Metabolite("h"),
+    Metabolite("atp"),
+    Metabolite("adp"),
+    Metabolite("h2o"),
+    Metabolite("pi"),
+]
+
+rxns = [
+    Reaction(
+        "HEX";
+        metabolites = Dict(
+            "atp" => -1.0,
+            "glc__D" => -1.0,
+            "g6p" => 1.0,
+            "adp" => 1.0,
+            "h" => 1.0,
+        ),
+    ),
+    Reaction("PGI"; metabolites = Dict("g6p" => -1.0, "f6p" => 1.0)),
+    Reaction(
+        "PFK";
+        metabolites = Dict(
+            "f6p" => -1.0,
+            "atp" => -1.0,
+            "adp" => 1.0,
+            "h" => 1.0,
+            "fdp" => 1.0,
+        ),
+    ),
+    Reaction("FBA"; metabolites = Dict("fdp" => -1.0, "dhap" => 1.0, "g3p" => 1.0)),
+    Reaction("TPI"; metabolites = Dict("dhap" => -1.0, "g3p" => 1.0)),
+    Reaction(
+        "GAPD";
+        metabolites = Dict(
+            "g3p" => -1.0,
+            "nad" => -1.0,
+            "pi" => -1.0,
+            "h" => 1.0,
+            "nadh" => 1.0,
+            "13dpg" => 1.0,
+        ),
+    ),
+    Reaction(
+        "PGK";
+        metabolites = Dict("13dpg" => -1.0, "adp" => -1.0, "atp" => 1.0, "3pg" => 1.0),
+    ),
+    Reaction("PGM"; metabolites = Dict("3pg" => -1.0, "2pg" => 1)),
+    Reaction("ENO"; metabolites = Dict("2pg" => -1.0, "h2o" => 1.0, "pep" => 1)),
+    Reaction(
+        "PYK";
+        metabolites = Dict(
+            "pep" => -1.0,
+            "adp" => -1.0,
+            "h" => -1.0,
+            "atp" => 1.0,
+            "pyr" => 1.0,
+        ),
+    ),
+    Reaction(
+        "LDH";
+        metabolites = Dict(
+            "pyr" => -1.0,
+            "nadh" => -1.0,
+            "h" => -1.0,
+            "nad" => 1.0,
+            "lac__D" => 1.0,
+        ),
+    ),
+]
+
+add_metabolites!(model, mets)
+add_reactions!(model, rxns)
+
+model
+
+# Load some thermodynamic data, ΔG'⁰ from eQuilibrator. Each key maps a reaction id to a
```

Comment:
link to equilibrator + package + preferably some notice about how to get the numbers would be cool.

name: `gibbs_free_energies`  (everything here is thermodynamic datum :D )